build.xml

```diff
@@ -60,6 +60,8 @@
             patches/7921.diff
             patches/7923.diff
             patches/7926.diff
+            patches/8036.diff
+            patches/8038.diff
```

Comment:
Please add a `-draft` suffix to the patch filenames so that it reminds us to track and update the final approved version as and when it happens.